testbed/lib/keyboard_test_page.dart

```diff
@@ -74,23 +79,29 @@ class _KeyboardTestPageState extends State<KeyboardTestPage> {
     }
 
     int keyCode;
+    String logicalKey;
+    String physicalKey;
     switch (event.data.runtimeType) {
       case RawKeyEventDataMacOs:
         final RawKeyEventDataMacOs data = event.data;
         keyCode = data.keyCode;
+        logicalKey = data.logicalKey.debugName;
+        physicalKey = data.physicalKey.debugName;
         break;
       // The Windows and Linux shells are still sending Android-encoded events.
       // Once Windows and Linux keyboard support is in place and the shells send
       // the correct type, this will need to be updated.
-      case RawKeyEventDataAndroid:
-        final RawKeyEventDataAndroid data = event.data;
+      case RawKeyEventDataLinux:
+        final RawKeyEventDataLinux data = event.data;
         keyCode = data.keyCode;
+        logicalKey = data.logicalKey.debugName;
+        physicalKey = data.physicalKey.debugName;
         break;
       default:
         throw new Exception('Unsupported platform ${event.data.runtimeType}');
     }
 
-    _addMessage('${isKeyDown ? 'KeyDown' : 'KeyUp'}: $keyCode');
+    _addMessage('''${isKeyDown ? 'KeyDown' : 'KeyUp'}: $keyCode \nLogical key: ${logicalKey}\nPhysical key: ${physicalKey}''');
```

Comment:
Why the change to triple `'`?

Reply:
I forgot, but i think i was trying to do multi-line string, but found out i don't need it.